models/salesforce/intermediate/int_salesforce__date_spine.sql

```diff
@@ -1,6 +1,11 @@
+{% if var('salesforce__lead_enabled', True) -%}
+-- depends_on: {{ var('lead') }}
+{% else -%}
+-- depends_on: {{ var('opportunity') }}
+{% endif %}  
```

Comment:
This is still necessary for `dbt run`. I tried running without it, and the error said to put it back. It's still not necessary for compilation, however.